main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	clientpkg "awscleaner/pkg/aws"
```

Comment:
This needs to be the full path to the pkg

`github.com/openshift/aws-account-shredder/pkg/aws`